src/index.html

```diff
@@ -4,15 +4,73 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__container">
+        <div class="card__product-photo">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+            class="card__img"
+          />
+        </div>
+
+        <h2 class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="reviews">Reviews: 5</p>
```

Comment:
```suggestion
          <p class="card__reviews">Reviews: 5</p>
```
It's not a separate BEM block

Reply:
Thank you for your valuable comments.